docs/configuration.rst

```diff
@@ -77,6 +77,15 @@ the migration is done and remove those two checksum types from the allowed list.
 the setting, you likely will need to run ``pulpcore-manager handle-artifact-checksums`` to remove
 unsupported checksums from the database, or Pulp will refuse to start.
 
+.. note::
+
+    If you'd like to run Pulp 3 on a machine where FIPS is enabled, you would need to patch
+    Django and Pulp at your own risk https://hackmd.io/@pulp/Pulp3-FIPS-matrix#Patches. The
+    patches will go away when Django project starts supporting FIPS mode.
```

Comment:
"...starts supporting FIPS mode (see https://code.djangoproject.com/ticket/28401 for details)."